compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/statements/ClientDeclarationStatementTest.java

```diff
@@ -1,97 +0,0 @@
-/*
```

Comment:
Relevant entry not removed the testng file?

Reply:
Already removed the `resources/statements/client-decl-stmt`